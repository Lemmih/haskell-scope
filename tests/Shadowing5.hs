module Shadowing4 where

x = case x of
      x -> x
  where
    x = 10
