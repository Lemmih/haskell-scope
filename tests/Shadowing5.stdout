Scope errors:

Definitions:
  Definition 1, from Shadowing4.decl0:
    [4mx[0m = case x of
  Definition 2, from Shadowing4.decl0.rhs.alt0:
          [4mx[0m -> x
  Definition 3, from Shadowing4.decl0.rhs.binds0:
        [4mx[0m = 10

Use sites:
  Definition used: 3
    x = case [4mx[0m of
  Definition used: 2
          x -> [4mx[0m

Exports:
  x, Value from Shadowing4
